sdk/playwrighttesting/microsoft-playwright-testing/src/common/messages.ts

```diff
@@ -10,4 +10,6 @@ export const ServiceErrorMessageConstants = {
   INVALID_GLOBAL_FUNCTION: "file must export a single function",
   DUPLICATE_REPORTER:
     "Another reporter with the same name already exists. Please resolve the duplicate and try again.",
+  INVALID_PLAYWRIGHT_VERSION_ERROR:
+    "The Playwright version you are using is not supported. Please upgrade to Playwright 1.47 or above.",
```

Comment:
We will need to update this error message every time in future

Reply:
@ShreyaAnand Not really this is one time

Reply:
v1.47 is the minimum playwright version required at least for sdk 1.x.x since it contains babel fix from OSS team. We don't expect this to change in the near future at least.